sklearn/preprocessing/data.py

```diff
@@ -397,6 +399,239 @@ def inverse_transform(self, X, copy=None):
         return X
 
 
+class RobustScaler(BaseEstimator, TransformerMixin):
+    """Scale features using statistics that are robust to outliers.
+
+    This Scaler removes the median and and scales the data according to
```

Comment:
and and

please go over the text one more time. I'll read again when you did so.